<?xml version="1.0" ?>
<launch>
    <arg name="robot_ip" default="172.16.0.2"/>
    <arg name="load_gripper" default="true" />
    <arg name="top_camera" default="true" />
    <arg name="arm_camera" default="true" />

    <group if="$(arg arm_camera)" ns="arm_camera">
        <include file="$(find realsense2_camera)/launch/includes/nodelet.launch.xml">
            <arg name="serial_no"         value="752112070781"/>
            <arg name="json_file_path"    value=""/>
            <arg name="depth_width"       value="640"/>
            <arg name="depth_height"      value="480"/>
            <arg name="depth_fps"         value="30"/>
            <arg name="color_width"       value="640"/>
            <arg name="color_height"      value="480"/>
            <arg name="color_fps"         value="30"/>
            <arg name="enable_depth"      value="true"/>
            <arg name="enable_color"      value="true"/>
            <arg name="enable_infra1"     value="false"/>
            <arg name="enable_infra2"     value="false"/>
            <arg name="enable_fisheye"    value="false"/>
            <arg name="enable_gyro"       value="false"/>
            <arg name="enable_accel"      value="false"/>
            <arg name="enable_pointcloud" value="true"/>
            <arg name="enable_sync"       value="true"/>
            <arg name="align_depth"         default="true"/>
            <arg name="tf_prefix"         value="top_camera"/>
            <arg name="publish_tf"               default="true"/>
            <arg name="tf_publish_rate"          default="0"/> <!-- 0 - static transform -->
        </include>
    </group>
    <group if="$(arg top_camera)" ns="top_camera">
        <include file="$(find realsense2_camera)/launch/includes/nodelet.launch.xml">
            <arg name="serial_no"         value="048122070681"/>
            <arg name="json_file_path"    value=""/>
            <arg name="depth_width"       value="640"/>
            <arg name="depth_height"      value="480"/>
            <arg name="depth_fps"         value="30"/>
            <arg name="color_width"       value="640"/>
            <arg name="color_height"      value="480"/>
            <arg name="color_fps"         value="30"/>
            <arg name="enable_depth"      value="true"/>
            <arg name="enable_color"      value="true"/>
            <arg name="enable_infra1"     value="false"/>
            <arg name="enable_infra2"     value="false"/>
            <arg name="enable_fisheye"    value="false"/>
            <arg name="enable_gyro"       value="false"/>
            <arg name="enable_accel"      value="false"/>
            <arg name="enable_pointcloud" value="true"/>
            <arg name="enable_sync"       value="true"/>
            <arg name="align_depth"         default="true"/>
            <arg name="tf_prefix"         value="top_camera"/>
            <arg name="publish_tf"               default="true"/>
            <arg name="tf_publish_rate"          default="0"/> <!-- 0 - static transform -->
        </include>
    </group>
</launch>